mlos_bench/mlos_bench/schedulers/cyclic_scheduler.py

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""A simple single-threaded synchronous optimization loop implementation."""
+import logging
+from typing import Any, Dict, List
+from mlos_bench.environments.base_environment import Environment
+from mlos_bench.optimizers.base_optimizer import Optimizer
+from mlos_bench.schedulers.sync_scheduler import SyncScheduler
+from mlos_bench.storage.base_storage import Storage
+from mlos_bench.util import merge_parameters
+
+_LOG = logging.getLogger(__name__)
+
+
+class CyclicScheduler(SyncScheduler):
+    """
+    A simple single-threaded synchronous benchmarking loop implementation
```

Comment:
```suggestion
    A simple synchronous benchmarking loop implementation
```

Reply:
Eventually, it'd be good to have a parallel version of this that can evaluate the default vs. new_suggestion across multiple nodes, so please also rename this to CyclicSyncScheduler.